Added Search Bar feature

nalvoj opened the pull request.

nalvoj committed:
Added Search Bar feature

aleskucera reviewed (changes requested):
I feel like this search bar may be free and easy to implement, but I find it rather difficult to find something in there. 

For example our common spot for experiments - Kateřinská zahrada:

- When I type Kateřinská zahrada, it takes me there without any other input (feels weird)
- When I type 'Katerinska zahrada' or 'Kateřinská zahr', nothing is found... 

I would prefer to switch to some other API, even if it would not be free.

nalvoj committed:
Revert "Added Search Bar feature"

This reverts commit 754956e173d8792da4ea22610b710817cbe49741.

nalvoj committed:
Beginning SearchBar feature

updated APIs, added SearchBar in html and css, granted functionality of SearchBar in map.js

nalvoj committed:
Done SearchBar

nalvoj committed:
Added replan button, made SearchBar collapsible

nalvoj commented:
Added SearchBar feature, added replan button

aleskucera reviewed (commented):
I don't have a `SEZNAM_API_KEY` so I cannot test new feature.
